spec/lib/terraforming/resource/iam_group_spec.rb

```diff
@@ -27,7 +27,15 @@ module Resource
       end
 
       before do
-        client.stub_responses(:list_groups, groups: groups)
+        client.stub_responses(:list_groups, [{
+                                               groups: [groups[0]],
+                                               is_truncated: true,
+                                               marker: 'marker'
```

Comment:
__[rubocop]__
* Align the elements of a hash literal if they span more than one line.